release-notes/Cube/Cube_Main_Release_10.4/Cube_10.4.0_features_changes.md

```diff
@@ -9,6 +9,57 @@ uid: Cube_Main_Release_10.4.0_other_features_changes
 
 ## Other new features
 
+#### Visual Overview: New BookingData component [ID_33215] [ID_36489]
+
+<!-- MR 10.4.0 - FR 10.3.8 -->
+
+You can now create a special *BookingData* shape and make it display all data associated with a particular booking.
+
+To do so, create a shape with the following shape data fields:
+
+|Shape data field | Value |
+|-------------|---------------|
+| Component   | `BookingData` |
+| Reservation | The ID of the booking<br>Example: `[pagevar:SRMRESERVATIONS_IDOfSelection]` |
+
+A *BookingData* shape will show the following information:
+
+- On the left-hand side, you will find a list of all resources in the booking.
+
+  For every resource, this list shows the following information:
+  
+  - the resource name
+  - an icon indicating the function of the resource
+  - the node label or, if no node label is defined, the name of the function definition
+  - the icon of the service definition node (only if the resource to mapped to a service definition node)
+
+- On the right-hand side, you can select a node or node interface of the resource selected in the list on the left.
+
+  After selecting a node or node interface, this section will show the profile data of this node or node interface:
+
+  - the profile instance, and
+  - the overridden parameter values.
```

Comment:
Note that these overridden values can be done on several levels.
So eventually, the user will see "the actual used parameter values of the profile parameters'.
Should this info be taken into account?